README.md

```diff
@@ -17,29 +20,37 @@ Once the project is cloned to disk you can import into Android Studio:
 * Navigate to the **arcgis-maps-sdk-kotlin-samples/** folder and click **OK**.
 
 ## Accessing Esri location services
+
 Accessing Esri location services, including basemaps, routing, and geocoding, requires authentication using either an API Key or an ArcGIS identity:
- 1. API key: A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
- 
-The Android samples in this repository have been structured to use an API key, set once, which will run in all samples. Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`). The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
 
-2. ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
+* API key
+
+A permanent key that gives your application access to Esri location services. Visit your [ArcGIS Developers Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.
+The Android samples in this repository have been structured to use an API key, set once, which will run in all samples.
+Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory (`/Users/<user_name>/.gradle/gradle.properties`).
+The API_KEY property should contain quotes around the key itself: `API_KEY = "YOUR_API_KEY"`
+
```

Comment:
```suggestion
Set your API key in the `gradle.properties` file located in the `/.gradle` folder within your home directory.
The API_KEY property should contain quotes around the key itself: 

```
Minor suggestion:
I think pulling out the `gradle.properties` example as a code block would help better clarify the usage for API key. Could add something like this below the API key description:
```gradle
API_KEY = "YOUR_API_KEY" // path: /Users/<user_name>/.gradle/gradle.properties
```